ext/vscode/resources/walkthrough/empty.md

```diff
@@ -0,0 +1 @@
+<!-- Empty Markdown document to facilitate walk-through steps without media -->
```

Comment:
You can delete this file if the extension doesn't contribute a walkthrough.

Reply:
We will need to add it, yet to push the files here.